tests/model/eventsources/test_msk_event_source.py

```diff
@@ -0,0 +1,46 @@
+from unittest import TestCase
+from samtranslator.model.eventsources.pull import MSK
+
+
+class MSKEventSource(TestCase):
```

Comment:
Just curious, any specific value to this test compared to a transform test? We typically prefer transform tests, as they mimic customer templates much closer, and are validated against [the JSON schema](https://github.com/aws/serverless-application-model/discussions/2645).

Reply:
This is just a unit test that I added to test my logic. I found a bunch of tests in that same folder for other event sources so I thought I should add one as well for the logic I added in MSK. I also added transform tests.

Reply:
I see, makes sense. 👍  In general if there are transform tests covering the transform logic, there's no need for separate unit tests.